images/scripts/bootc.setup

```diff
@@ -27,6 +27,14 @@ podman rmi localhost:5000/bootc:latest localhost/bootc:latest
 podman rm -f -t0 ostree-registry
 rm /var/cache/bootc.oci.tar
 
+# Older RHEL releases need vsock support for test.thing
+cp -va /var/lib/testvm/test.thing-workarounds/*.{socket,service} /etc/systemd/system
+systemctl daemon-reload
+systemctl enable tt-sshd-vsock.socket tt-sd_notify.service
+checkmodule -M -m -o /tmp/sshd_vsock.mod /var/lib/testvm/test.thing-workarounds/sshd_vsock.te
+semodule_package -m /tmp/sshd_vsock.mod -o /tmp/sshd_vsock.pp
+semodule -i /tmp/sshd_vsock.pp
+
```

Comment:
I know we don't have a centos-10-bootc yet (we actually should..). Can you please restrict that to rhel-9 like you did on the rpm image? Then we'll also see exactly when to drop it, and confirm that RHEL 10 doesn't need that.